<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileDownloadDialog</class>
 <widget class="QWidget" name="FileDownloadDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>858</width>
    <height>437</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FileDownloadDialog</string>
  </property>
  <widget class="QPushButton" name="btnOK">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>350</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Download</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblMessage">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>50</y>
     <width>401</width>
     <height>141</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Calibration datas are needed.
 If you want to download calibration data, please press the Ok button</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBarFile1">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>272</y>
     <width>661</width>
     <height>31</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="textVisible">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lblFile1">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>270</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Calibration Data</string>
   </property>
  </widget>
  <widget class="QLabel" name="statusLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>360</y>
     <width>56</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
